kv/db.go

```diff
@@ -163,57 +163,57 @@ func (s *rpcDBServer) executeCmd(args proto.Request, reply proto.Response) error
 	return nil
 }
 
-// Get .
+// Get.
```

Comment:
These doc comments shouldn't be necessary on an unexported type. They are sometimes used on exported types to silence golint. Correct me if I'm mistaken here and golint is asking for these, but if it isn't I'd prefer not to see them.